api/v1alpha08/sonataflow_persistence_types.go

```diff
@@ -14,6 +14,32 @@
 
 package v1alpha08
 
+// PlatformPersistenceOptionsSpec configures the DataBase in the platform spec. This specification can
+// be used by workflows and platform services when they don't provide one of their own.
+// +optional
+// +kubebuilder:validation:MaxProperties=1
```

Comment:
nit - maybe add notes to these persistence struct descriptions regarding the addition (or subtraction) of the `databaseSchema` field?

Reply:
Isn´t it a new struct? The original one still remains unchanged from the API point of view. So mentioning that it doesn´t have the `databaseSchema` when it never had one sounds a bit strange to me. Maybe I'm missing something here.